src/test_typing_extensions.py

```diff
@@ -5605,6 +5605,23 @@ def test_typevartuple(self):
         class A(Generic[Unpack[Ts]]): ...
         Alias = Optional[Unpack[Ts]]
 
+    def test_erroneous_generic(self):
+        DefaultStrT = TypeVar('DefaultStrT', default=str)
```

Comment:
A bit weird that this one uses plain TypeVar while the next test uses `typing_extensions.TypeVar`. They are the same (we do `from typing_extensions import TypeVar`), so we should stick to just one variation. I think it is clearer to use `typing_extensions.TypeVar` but don't feel strongly.

(No need to change this in other tests.)